sdk/rust/oak/src/handle.rs

```diff
@@ -0,0 +1,102 @@
+include!(concat!(env!("OUT_DIR"), "/oak.handle.rs"));
+
+use crate::OakError;
+
+pub type Handle = oak_abi::Handle;
```

Comment:
Please do not re-export unless it is necessary.

Reply:
I think we have to in this case, because crates that depend on `oak` maybe not explicitly add `oak_abi` as a dependency, and we need to have this `Handle` type defined for the derive macro. Alternatively we could just have the macro use `u64`, but that feels like relying on implementation details.

Reply:
If that's the case, please add a comment :)

Reply:
Done.